apps/src/templates/rubrics/EvidenceLevelsForTeachersV2.jsx

```diff
@@ -0,0 +1,98 @@
+import React, {useMemo} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {aiEvaluationShape, evidenceLevelShape} from './rubricShapes';
+import {
+  BodyThreeText,
+  BodyFourText,
+  StrongText,
+  Heading6,
+} from '@cdo/apps/componentLibrary/typography';
+import {
+  UNDERSTANDING_LEVEL_STRINGS,
+  UNDERSTANDING_LEVEL_STRINGS_V2,
+} from './rubricHelpers';
+
+export default function EvidenceLevelsForTeachersV2({
+  evidenceLevels,
+  understanding,
+  radioButtonCallback,
+  canProvideFeedback,
+  isAutosaving,
+  aiEvalInfo,
+}) {
+  const passFail = useMemo(() => {
```

Comment:
I wonder if we could name this a bit more descriptively... `selectPassOrFailOptions`?  I don't love that, but I am just trying to think of what might make this a bit more descriptive to a new reader...

Reply:
I added some comments to describe the element.